modules/builtin/src/actions/switchLanguage.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Update user language
+ * @title Update user language
+ * @category Language
+ * @author Botpress, Inc.
+ * @param {string} lang - The language code, e.g. "en"
+ */
+const myAction = async lang => {
+  event.state.user.language = lang
+  await bp.users.updateAttributes(event.channel, event.target, { language: lang })
+  await event.setFlag(bp.IO.WellKnownFlags.FORCE_PERSIST_STATE, true)
```

Comment:
Calling updateAttributes is redundant, since the state manager will do the same with FORCE_PERSIST_STATE

Reply:
Done in https://github.com/botpress/botpress/pull/2392/commits/2cd9b66fe1bce69cc9d4faa37023eea7b6e78553